packages/jobs/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["./node_modules/@latitude-data/eslint-config/library.js"],
+  "env": {
+    "node": true,
+    "browser": true
+  },
+  "rules": {
+    "no-constant-condition": "off"
+  }
+}
```

Comment:
we are using eslint 8 across all our projects because this json syntax is simpler